src/test/java/utilities/TaskListTest.java

```diff
@@ -0,0 +1,33 @@
+package utilities;
+
+import tasks.ToDos;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import org.junit.jupiter.api.Test;
+
+// Assumes the data file is based on the default generated by runtest
+public class TaskListTest {
+    private static final String FILE_PATH = "tasks/data.txt";
+    private static final String ACTUAL_STRING = "1.[T][ ]  borrow book \n"
+            + "2.[E][X]  project meeting from: Dec 29 2024 12:00 to: Dec 29 2024 17:00 \n"
+            + "3.[T][ ]  borrow notes";
+    private Storage store = new Storage(FILE_PATH);
```

Comment:
I think there can be a space above this line as this is separate from the 2 constants above.